code-challenges.test.js

```diff
@@ -58,3 +125,21 @@ var numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+// let newNums = numbersToAdd1.filter(value => {
+//     return value !== 0
+//     })
+//     let newNums2 = numbersToAdd2.filter(value => {
+//         return value !== 0
+//         })
+//         let newNums3 = numbersToAdd3.filter(value => {
+//             return value !== 0
+//             })
+    const numAdder = (sum => value => sum += value)(0);
+    
+    console.log(numbersToAdd1.map(numAdder));
+    console.log(numbersToAdd2.map(numAdder));
+    console.log(numbersToAdd3.map(numAdder));
```

Comment:
The fact that you have console.logs here means I know you aren't using jest to test your answers. Please use jest to test your answers and logs as you go!